examples/randomWalk.js

```diff
@@ -11,13 +11,33 @@ function stepFunc() {
     return pg.geometry.toCircle(2 * Math.PI * Math.random())
 }
 
+function evenCircleStart(numPoints, radius){
+    return pg.geometry.circlePoints(numPoints, radius);
+}
+
+function archSpiralStart(a, b, n, pointsPerCycle){
+    var startPoints = Array(n);
+    var theta = 0;
+    for(var i = 0; i < n*pointsPerCycle; i++){
+        theta += 2*Math.PI/pointsPerCycle;
+        var r = a + b*theta;
+        var point = [0,0];
+        point[0] = pg.geometry.toCircle(theta)[0]*r;
+        point[1] = pg.geometry.toCircle(theta)[1]*r;
```

Comment:
was getting weird errors, not sure if this was or wasn't the cause. 
I will try switching back.